third_party/2and3/flask/templating.pyi

```diff
@@ -17,5 +17,8 @@ class DispatchingJinjaLoader(BaseLoader):
     def get_source(self, environment: Any, template: Any): ...
     def list_templates(self): ...
 
-def render_template(template_name_or_list: Any, **context: Any): ...
-def render_template_string(source: Any, **context: Any): ...
+def render_template(
+    template_name_or_list: Union[str, Iterable[str]],
```

Comment:
Are you sure this is str rather than Text? (Does it accept unicode in Python 2?)

Reply:
I just checked it out, works with unicode, will update.